src/System.Windows.Forms/src/System/Windows/Forms/Form.cs

```diff
@@ -4901,6 +4908,27 @@ protected override void ScaleCore(float x, float y)
             }
         }
 
+        protected override void ScaleTopLevelWindowProperties(float widthFactor, float hightFactor)
```

Comment:
```suggestion
        protected override void ScaleTopLevelWindowProperties(float widthFactor, float heightFactor)
```
Though I'd suggest the following as we generally use `x` and `y`:
```suggestion
        protected override void ScaleTopLevelWindowProperties(float xScaleFactor, float yScaleFactor)
```